src/OpenTelemetry.Instrumentation.Http/README.md

```diff
@@ -62,14 +65,48 @@ public class Program
 }
 ```
 
-For an ASP.NET Core application, adding instrumentation is typically done in the
-`ConfigureServices` of your `Startup` class. Refer to documentation for
-[OpenTelemetry.Instrumentation.AspNetCore](../OpenTelemetry.Instrumentation.AspNetCore/README.md).
+#### Metrics
+
+The following example demonstrates adding HTTP client instrumentation with the
+extension method `.AddHttpClientInstrumentation()` on `MeterProviderBuilder` to
+a console application. This example also sets up the OpenTelemetry Console
+Exporter, which requires adding the package
+[`OpenTelemetry.Exporter.Console`](../OpenTelemetry.Exporter.Console/README.md)
+to the application.
+
+```csharp
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+public class Program
```

Comment:
Instead of using the explicit class and Main entry point, we might want to shift to a simpler model https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/getting-started-aspnetcore/Program.cs. Doesn't block this PR though.